operator/src/test/java/oracle/kubernetes/operator/helpers/HealthCheckHelperTest.java

```diff
@@ -117,37 +116,22 @@ public void whenRulesReviewSupported_accessGrantedForEverything() {
     expectSelfSubjectRulesReview();
 
     for (String ns : TARGET_NAMESPACES) {
-      V1SubjectRulesReviewStatus status = HealthCheckHelper.getSelfSubjectRulesReviewStatus(ns);
-      HealthCheckHelper.verifyAccess(status, ns, true);
+      HealthCheckHelper.getAccessAuthorizations(ns);
     }
   }
 
   @Test
-  public void whenRulesReviewSupportedAndNoDomainNamespaceAccess_logWarning() {
+  public void whenRulesReviewSupportedAndNoNamespaceAccess_logWarning() {
     accessChecks.setMayAccessNamespace(false);
     expectSelfSubjectRulesReview();
 
     for (String ns : TARGET_NAMESPACES) {
-      V1SubjectRulesReviewStatus status = HealthCheckHelper.getSelfSubjectRulesReviewStatus(ns);
-      HealthCheckHelper.verifyAccess(status, ns, true);
+      HealthCheckHelper.getAccessAuthorizations(ns);
     }
 
     assertThat(logRecords, containsWarning(VERIFY_ACCESS_DENIED_WITH_NS));
   }
 
-  // HERE
```

Comment:
Whoops! I extraneous "HERE" message was checked-in, but otherwise this is another accidental revert of a real change.

Reply:
Think I have it fixed